llvm/test/MC/ELF/warn-newline-in-escaped-string.s

```diff
@@ -0,0 +1,54 @@
+// RUN: llvm-mc -filetype=obj -triple x86_64 %s 2>&1 -o /dev/null | FileCheck -DFILE=%s --strict-whitespace %s --implicit-check-not="{{[0-9]+:[0-9]+: warning: unterminated string}}" --check-prefix=CHECK-WARN
```

Comment:
```suggestion
// RUN: llvm-mc -filetype=obj -triple x86_64 %s 2>&1 -o /dev/null \
//   | FileCheck %s --implicit-check-not="{{[0-9]+:[0-9]+: warning: unterminated string}}" --check-prefix=CHECK-WARN
```

-DFILE is unused and --strict-whitespace no longer seems necessary (?)
Long RUN lines can be split into multiple by the backslash character.

Reply:
Done